avocado/utils/build.py

```diff
@@ -10,17 +10,18 @@
 # See LICENSE for more details.
 #
 # Copyright: Red Hat Inc. 2013-2014
+#          : IBM 2016
 # Author: Lucas Meneghel Rodrigues <lmr@redhat.com>
+#       : Praveen K Pandey <praveen@linux.vnet.ibm.com>
```

Comment:
See eg. the first response to http://programmers.stackexchange.com/questions/203014/author-has-inserted-copyright-into-code-with-gnu-public-license-notice-implica